### ISSUE TITLE
Senha do Certificado

### ISSUE THREAD

**Issue by matheusfols** [DenysXavier/WSBoletoSantander#44]
Olá,

Onde consigo a senha do certificado?

**Comment by DenysXavier** [comment]
Oi, @matheusfols!

A senha do certificado é um segredo.

Se você não criou o certificado ou se seu cliente não se lembra qual a senha pra te informar, então, será necessário fazer um novo certificado, pois não terá como abrí-lo.

**Comment by matheusfols** [comment]
@DenysXavier obrigado pela resposta.

O certificado dele é gratuito fornecido pelo servidor de hospedagem. Nesse caso teria que gerar um novo msm?!

**Comment by DenysXavier** [comment]
Olá, @matheusfols!

Boa pergunta. Nunca peguei um caso assim.

Se tiver que chutar, eu diria que sim. Pois, geralmente, esses certificados gratuitos de servidor de hospedagem são compartilhados e eles não fornecem a senha.

Além do mais, antes mesmo de você conseguir usar essa biblioteca, você terá que registrar seu certificado no Santander, caso contrário, eles nem aceitam sua conexão.

**Comment by matheusfols** [comment]
@DenysXavier pior que já enviei o certificado e foi aprovado.
Vou ter que ver outro para o cliente.

**Comment by matheusfols** [comment]
@DenysXavier vi que tem uns comandos de conversão de cer para pem, é possível inserir senha nesse momento, sabe me dizer?

**Comment by DenysXavier** [comment]
@matheusfols,

Sim, é possível inserir senha nesse momento. Contudo, você ainda precisará da senha do arquivo que você está tentando converter.

Acho que isso não resolve seu problema.
